### ISSUE TITLE
Use of maxResults setting in JenaTextSparql has no effect

### ISSUE THREAD

**Issue by rwalkerands** [NatLibFi/Skosmos#794]
I am using the latest release of Fuseki 2. Perhaps this issue did not arise in earlier releases (?).

In `JenaTextSparql.php`, the use of the `MAX_N` setting in `createTextQueryCondition()` to "limit" the number of search results has no effect.

Refer to the Jena documentation at https://jena.apache.org/documentation/query/text-query.html:

> The most general form is:
> `( ?s ?score ?literal ?g ) text:query ( property 'query string' limit 'lang:xx' 'highlight:yy' )`

and very specifically:

> If both `limit` and one or more of `lang:xx` or `highlight:yy` are present, then `limit` must precede these arguments.

But in `JenaTextSparql.php` there is this line:
```
return "(?s ?score ?match) text:query ($property '$term' $langClause $maxResults) .";
```
which puts the limit _after_ the language.

## What steps will reproduce the problem?
1. Configure a `skosmos:Vocabulary` with its SPARQL endpoint hosted in Fuseki, and ensure that the property `skosmos:sparqlDialect "JenaText"` is set for it. Ensure that the vocabulary has a few concepts in it with prefLabels that begin with `A` (let's say, more than 5).
2. View the "front page" of the vocabulary. Observe the contents of the "A-Z" tab. This _should be_ a convenient way to see the effect of the `MAX_N` setting, as this tab shows the results for a search on `A*`.
3. Edit `JenaTextSparql.php`, and change the setting of `MAX_N` to a very small value, e.g., 5.
4. Refresh the "front page" of the vocabulary. Observe the "A-Z" tab. There is no change, but there _should_ now only be 5 concepts listed.

To have the setting take effect as it was supposed to:

5. Edit `JenaTextSparql.php` again and change the line:
```
return "(?s ?score ?match) text:query ($property '$term' $langClause $maxResults) .";
```
to:
```
return "(?s ?score ?match) text:query ($property '$term' $maxResults $langClause) .";
```
6. Refresh the "front page" of the vocabulary. Observe the "A-Z" tab. There are now only 5 concepts listed.

**Comment by osma** [comment]
Thanks again @rwalkerands for the report and excellent analysis!

The MAX_N constant was added because we found that the default result limit in jena-text is sometimes too small, so some results may be omitted seemingly at random.

Jena-text has grown over time with new features such as the restriction by language. It seems likely that the bug happened while adding support for the language clause in Skosmos. The fix is easy - you've already demonstrated it!
